sig-testing/annual-report-2023.md

```diff
@@ -4,38 +4,31 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+Crossover with SIG K8s Infra. Migration of CI (mainly jobs, AWS build cluster,
+etc.) to community-owned infra, which is also a prerequisite for migrating the
+Prow control plane in 2024. See the umbrella issue at
+[k8s/test-infra#29722](https://github.com/kubernetes/test-infra/issues/29722).
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
-<!--
-   Note: This list is generated from the KEP metadata in kubernetes/enhancements repository.
-      If you find any discrepancy in the generated list here, please check the KEP metadata.
-      Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
--->
+- [prow](https://sigs.k8s.io/prow/)
+- [testgrid](https://sigs.k8s.io/testgrid/)
+- [boskos](https://sigs.k8s.io/boskos/)
+- [kettle](https://git.k8s.io/test-infra/kettle/)
+- [kubetest2](https://sigs.k8s.io/kubetest2/)
```

Comment:
If there are fewer than 2 active OWNERS, how many active owners are there? 1 or 0?

BTW, do we have a priority for those projects? 
@BenTheElder

Reply:
My estimate (someone else feel free to weigh in):

- prow: 1 active owner?
- testgrid: 1 active owner (me)
- boskos: 1 active owner (krzyzacy)
- kettle: unsure. SIG K8s Infra recently migrated this, but for the actual listed owners this is either 0, or 1 (MushuEE)
- kubetest2:  ~0 to my knowledge.